.github/workflows/ci.yaml

```diff
@@ -12,16 +12,16 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.x']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
Happy to test the most recent python versions here! We probably want to include a note in the changelog as well.

On the other hand, I think we want to continue testing `3.7` before it's been properly removed from the codebase, see the relevant ticket here: https://github.com/beetbox/beets/issues/5222